docs/api-reference/connect-to-log.md

```diff
@@ -1,8 +1,8 @@
 # connectToLog
 
 Wraps a React component in a
```

Comment:
This should probably get reverted? It looks like you just accidentally made a change.

Reply:
This is was auto-changed by the linter.